### ISSUE TITLE
Unrecognized syntaxes

### ISSUE THREAD

**Issue by Romitou** [ShaneBeee/SkBoard#1]
Hello, thank you very much for this addon at first.
So I wanted to test it, but I did not succeed because the syntaxes are not recognized in my case.

I used the documentation and tried to put the scoreboard for all the players. Here is my code:
```
function scoreboard():
	set title of all player's scoreboards to "Hello."
	set line 10 of all player's scoreboards to "How are you?"
```

MC: `Paper version git-Paper-112, API 1.15.2-R0.1-SNAPSHOT`
SK: `2.5-alpha1` (yes, I should update ;p)
SkBoard: `1.0.1`

I hope that the error does not come from me, I am sorry to have opened a issue for nothing in this case.

Hoping to have provided all the necessary information and that the error does not come from me,
Have a good day !

**Comment by ShaneBeee** [comment]
> Hello, thank you very much for this addon at first.

You're welcome ;)

As for the code:
all players (with anything in Skript) is done as `all players'`

**Comment by Romitou** [comment]
Thank you for your answer, I thought it was an error in the documentation!
So I tested with the right syntaxes, even trying to remove the `'`, but it still doesn't work with this code:

```
function scoreboard():
	toggle scoreboards of all players on # test syntax, but recognized
	set title of all players' scoreboards to "Hello." # still does not work
	set title of all players scoreboards to "Hello." # still does not work
	set line 10 of all players' scoreboards to "How are you?" # still does not work
	set line 10 of all players scoreboards to "How are you?" # still does not work
```
Here are the errors (oops, I forgot to put them in my first post):
```
Can't understand this condition/effect: set title of all players' scoreboards to "Hello." (others.sk, line 24: set title of all players' scoreboards to "Hello."')
Can't understand this condition/effect: set title of all players scoreboards to "Hello." (others.sk, line 25: set title of all players scoreboards to "Hello."')
Can't understand this condition/effect: set line 10 of all players' scoreboards to "How are you?" (others.sk, line 26: set line 10 of all players' scoreboards to "How are you?"')
Can't understand this condition/effect: set line 10 of all players scoreboards to "How are you?" (others.sk, line 27: set line 10 of all players scoreboards to "How are you?"')
```

**Comment by Romitou** [comment]
However, the single player syntaxes work, the error must surely come from me. :sweat_smile:

**Comment by ShaneBeee** [comment]
oh sorry, I did make a mistake in the syntax.
I forgot to make the last `s` optional, so til I fix it you'll have to do `all players's`

**Comment by ShaneBeee** [comment]
Fixed in 1.0.2
